moto/dms/models.py

```diff
@@ -0,0 +1,200 @@
+from __future__ import unicode_literals
+
+import json
+
+from boto3 import Session
+from datetime import datetime
+from moto.core import ACCOUNT_ID, BaseBackend, BaseModel
+
+from .exceptions import (
+    InvalidResourceStateFault,
+    ResourceAlreadyExistsFault,
+    ResourceNotFoundFault,
+)
+from .utils import filter_tasks
+
+
+class DatabaseMigrationServiceBackend(BaseBackend):
+    def __init__(self, region_name=None):
+        super(DatabaseMigrationServiceBackend, self).__init__()
+        self.region_name = region_name
+        self.replication_tasks = {}
+
+    def reset(self):
+        region_name = self.region_name
+        self.replication_tasks = {}
```

Comment:
This shouldn't be necessary - resetting with `self.__dict__ = {}` already wipes all data